imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -37,3 +41,17 @@ func TestGuestReboot(t *testing.T) {
 	// second boot
 	t.Log("marker file exist signal the guest reboot successful")
 }
+
+func TestGuestSecureBoot(t *testing.T) {
+	if _, err := os.Stat(secureBootFile); os.IsNotExist(err) {
+		t.Skip("not supported on non-uefi boot disk")
```

Comment:
it is not a skip, it is a fail

Reply:
to elaborate, if this test were run against all VMs, skip would be fine here. but we specifically set up a VM to run this test against, and if the entire sysfs var is missing, that's a fail.

Reply:
I assume the test run against al lvm. but it's good to consider the case that efi var is missing.